### PR TITLE
Writing README.md

### PR DESCRIPTION
**Pull request by khc196**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,25 @@
-# alpha-search
\ No newline at end of file
+# Alpha Search
+## Authors
```
Comment (shberry) [inline README.md]: How about 'Authors and owners'?

File: README.md [hunk 2]
```diff
@@ -1 +1,25 @@
-# alpha-search
\ No newline at end of file
+# Alpha Search
+## Authors 
+*Minjun Koo, Gaeul Go, Hwancheol Kang, Kihyun Yang*<br>
+*- Kookmin University Computer Science*
+
+## Overview
+
+Alpha-Search is a chatbot-style recommendation system for users who are looking for multiple cafes for users who are looking for coffee shops where they can bring their laptop to do things. If a user asks chatbot to find a cafe that has user wants (e.g. Table shape , WiFi, Enough of a lot of outlet(number of outlets / number of seats > t : t is an arbitrary number to be determined later), etc.), Chatbot will give you a couple of candidates on the map based on the data stored in the DB . When users click on a candidate provided with a pin on map, detailed information about the place is provided and the user makes a decision among the candidates based on this information. Then, the chatbot provides a route to the selected location. In addition, the chatbot asks the user what they may need in addition to what the user has requested and use this information to determine the candidates.
```
Comment (shberry) [inline README.md]: "for users who are looking for coffee shops" -> redundant?
"that has user wants" -> how about "with certain features"?
"Table" -> "table". Same with "Enough"
"Enough of a lot of outlet" -> how about "enough outlets"?

How about we mention that's in the DB? I'm sure populating the DB is part of your job too :)

File: README.md [hunk 3]
```diff
@@ -1 +1,25 @@
...
+
+## Flowchart
+
+![flowchart](./flow_chart.png)
+
+## Infrastructure
+Server Infra : AWS EC2<br>
```
Comment (shberry) [inline README.md]: Infra -> Infrastructure, or just remove 'Infra'

**Review by shberry — COMMENTED** [review]
Good job! We can make this better :)

**Comment by khc196** [comment]
@shberry Thank you!! I fixed what you said. 👍

**Comment by mjkoo711** [comment]
It's good! Perfrct!

**Pull request merged**